README.md

```diff
@@ -234,3 +234,37 @@ Similarly, after adding or removing Go source files, it may be necessary to sync
 ```bash
 bazel run //:gazelle
 ```
+
+## Using this repository from other Bazel workspaces
+
+To use `go-jsonnet` from a different Bazel workspace, add the repository
+and its dependencies to your WORKSPACE file.
+
+You will also need to register a Go toolchain.
```

Comment:
Do you only need to do this if you don't already have one?

Reply:
That's right.